deps/buildmod_LibVPX.v1.8.0.jl

```diff
@@ -0,0 +1,49 @@
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
+
+# Parse some basic command-line arguments
+const verbose = "--verbose" in ARGS
+const prefix = Prefix(get([a for a in ARGS if a != "--verbose"], 1, joinpath(@__DIR__, "usr")))
+products = [
```

Comment:
You could conditionally set it to the proper value or to the empty one, depending on current operating system (with iswindows). You could also just upload this to libvpx release

Reply:
Good idea! And yeah, the buildmod files were temporary. I just uploaded it to the release